### PR TITLE
Ipkp 47 add crud to feed clean code

### PR DESCRIPTION
**Pull request by pavelb2p**
After squash

### CODE REVIEW

**Comment by pavelb2p** [comment]
Done:
- All correct imports
- Squash all commits
- Returns empty list if Feeds or Users is empty in Controllers
- Added feeds and users ID in method parameter
- Made private final field in Controller and ServiceImpl classes for Feed\UserService and Feed\UserRepository
- Cleaned all comments in code

File: pom.xml [hunk 1]
```diff
@@ -92,4 +128,5 @@
         <finalName>mycrudproject</finalName>
```
Comment (kastyan-kg) [inline pom.xml:128]: my_crud_project ot my-crud-project

File: pom.xml [hunk 2]
```diff
@@ -92,4 +128,5 @@
...
     </build>
 
+
```
Comment (kastyan-kg) [inline pom.xml:131]: extra space

**Review by kastyan-kg — APPROVED** [review]
some minor comments

**Pull request merged**